.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -0,0 +1,120 @@
+name: rawls-run-azure-e2e-tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch of rawls to run tests on'
+        required: true
+        default: 'develop'
+        type: string
+
+env:
+  BEE_NAME: '${{ github.event.repository.name }}-${{ github.run_id }}-${{ github.run_attempt}}-dev'
+  TOKEN: '${{ secrets.BROADBOT_TOKEN }}' # github token for access to kick off a job in the private repo
+
+jobs:
+  rawls-build-tag-publish-job:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    outputs:
+      custom-version-json: ${{ steps.render-rawls-version.outputs.custom-version-json }}
+    steps:
+      - uses: 'actions/checkout@v3'
+
+      - name: Bump the tag to a new version
```

Comment:
Rawls image tags are typically just the commit hash, using this seems like it'd be inconsistent

Reply:
otoh, semver is nice so maybe this is the first step towards moving Rawls over?